pkg/apiclient/workflow/workflow.proto

```diff
@@ -89,6 +89,7 @@ message WorkflowDeleteResponse {
 message WatchWorkflowsRequest {
     string namespace = 1;
     k8s.io.apimachinery.pkg.apis.meta.v1.ListOptions listOptions = 2;
+    string fields = 3;
```

Comment:
Where is this field actually used? I don't see it in the Argo Server code

https://github.com/rbreeze/argo/blob/162f49559cb4a24eadccb20129b8ce04fb36faca/server/workflow/workflow_server.go#L128

Reply:
In fact I don't see it used anywhere in List either: https://github.com/rbreeze/argo/blob/162f49559cb4a24eadccb20129b8ce04fb36faca/server/workflow/workflow_server.go#L94

Seems like that was the bug?